src/main/java/com/elixr/poc/rest/response/ExceptionHandler.java

```diff
@@ -0,0 +1,41 @@
+package com.elixr.poc.rest.response;
+
+import com.elixr.poc.rest.request.UserRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+Handling the exception and sending propper error message
+ */
+@ControllerAdvice
+public class ExceptionHandler {
+    private ErrorResponse errorResponse;
+
```

Comment:
Remove all empty lines like this from all files. Can keep one empty line for code readability, but multiple empty lines are not recommended

Reply:
removed all empty lines